blocks/table/table.css

```diff
@@ -0,0 +1,134 @@
+/* stylelint-disable no-descending-specificity */
+.table-container {
+    margin-bottom: 32px;
+}
+
+.table-container > div.table-wrapper {
+    margin-bottom: 0;
+}
+
+.table-container > .table-wrapper + .table-wrapper {
+    margin-top: 0;
+}
```

Comment:
the more i think about this, i think we should find a way to leverage table (as an element) more, where each table becomes a tbody inside a larger table, rather than three tables that inexorably linked being separate.

Reply:
I rewrote how we are building the table in table.js to address this issue you mention. Working adjusting the css to match the changes/simplify things.